lib/bamboo/adapters/send_grid_adapter.ex

```diff
@@ -161,6 +166,16 @@ defmodule Bamboo.SendGridAdapter do
 
   defp put_reply_to(body, _), do: body
 
+  defp put_headers(body, %Email{headers: headers}) when is_map(headers) do
+    headers_without_tuple_values =
+      Map.delete(headers, "reply-to")
+      |> Map.delete("Reply-To")
```

Comment:
Tiny code style thing. My parallel structure brain would find it easier to understand at a glance if it were written like: 
```
headers
|> Map.delete("reply-to")
|> Map.delete("Reply-To")
```

What do you think?

Reply:
Thanks @maymillerricci, I think that looks better, is more consistent with the code style, and feels more idiomatic. :alembic: :+1: